e2e/stress/stress.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28803.352
```

Comment:
Why is there a version degrade?

Reply:
Oh, you know, this is done automatically by Visual Studio. I'll revert this change.